examples/config_example.yaml

```diff
@@ -0,0 +1,15 @@
+# Config file for Z HMC OS Message Forwarder
+
+hmc:
+  hmc: 10.11.12.13
+  userid: "myuser"
+  password: "mypassword"
```

Comment:
Will there be a way to pass in the password via command line argument or environment variable?

Reply:
We need to do something about passwords in config files. In the most advanced form that would a user-extensible plug interface for a vault file, with a default implementation provided by us for a local vault file.
Anyone is welcome to come up with a concept like that. It should be in the zhmcclient library so we can reuse it in all packages on top of it.
But I don't want to start this now and here.